CHANGELOG.md

```diff
@@ -13,13 +13,13 @@
 
 - API: Change media type of GeoJSON response ([#199])
 - Validate indicator-layer combination with `pydantic` ([#190])
-- Improve API response when an error occurs by including informations about the cause ([#186])
+- Improve API response when an error occurs by including information about the cause ([#186])
 - Add parameter to get API response without svg string in the result. ([#137])
 - Extend time range for Currentness indicator ([#178])
 
 ### How to upgrade
 
-- Update requests for `/indicator` and `/report` endpoints. `name` is now a *Query* parameter ([#190])
+- Update requests for `/indicator` and `/report` endpoints. `name` is now a Query parameter ([#190])
```

Comment:
Opted for "Query" (no italics) to be consistence with previous use of the parameter name